### DISCUSSION TITLE
App Not Opening on External Drive/Nas Storage Drive

### DISCUSSION THREAD

**Discussion by iamLazyCode** [mifi/lossless-cut#2143] [General]
hello everyone i am facing a strange issue its not related to cutting the video if i keep lossless cut folder on widows the exe opens fine but as soon as i transfer it on external drive or nas storage drive the app doesnt opens

**Comment by mifi** [comment]
hi, looks like maybe the same as this #1893

**Reply by iamLazyCode** [reply]
thanks alot il give a read

**Reply by iamLazyCode** [reply]
> hi, looks like maybe the same as this #1893

it does work with given solution  , i hope in future it can be sorted with just simple click

![image](https://github.com/user-attachments/assets/c061d1e4-e13c-4423-8d68-91a147036516)
